mentat/session.py

```diff
@@ -84,6 +76,9 @@ def __init__(
         )
         SESSION_CONTEXT.set(session_context)
 
+        # Functions that require stream
```

Comment:
Requiring the stream is the same as requiring the session_context, so I think this would fit under the other comment